### ISSUE TITLE
Program crashes with segmentation fault upon launch.

### ISSUE THREAD

**Issue by xiraix** [cs01/gdbgui#353]
**Describe the bug**
Program crashes with segmentation fault upon launch.

```
~ > gdbgui
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
Opening gdbgui with default browser at http://127.0.0.1:5000
View gdbgui dashboard at http://127.0.0.1:5000/dashboard
exit gdbgui by pressing CTRL+C
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
zsh: segmentation fault (core dumped)  gdbgui

```

**To Reproduce**
Install gdbgui using pipx or pip.
Run it.

**Expected behavior**
gdbgui runs successfully 

**Please complete the following information:**
* OS: Linux archlinux 5.8.12-arch1-1
* gdbgui version (`gdbgui -v`): 0.14.0.0
* gdb version (`gdb -v`):  9.2
* python version: Python 3.8.5
* browser [e.g. chrome, safari]: firefox
* python packages (`pip freeze`): https://p.teknik.io/oeMdt

**Comment by maximilianwulf** [comment]
Also happens on Ubuntu 20.04.

**Comment by GCrispino** [comment]
Got the same problem. The following is printed in the terminal:
```
❯ gdbgui
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
Opening gdbgui with default browser at http://127.0.0.1:5000
View gdbgui dashboard at http://127.0.0.1:5000/dashboard
exit gdbgui by pressing CTRL+C
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
zsh: segmentation fault (core dumped)  gdbgui
```
When I first tried it did not segfaulted until I tried opening the dashboard.

I've got the same config as @xiraix, except for the OS (I got Manjaro 20.1), and maybe for the pip installed packages (pip freeze at https://pastebin.com/QzkWEZmC).

Nonetheless, I was able to run gdbgui and open the dashboard without errors after downloading the binary in the repository's releases page.

**Comment by SondreWang** [comment]
Got the same problem. It may be that something is incompatible with greenlet v. 0.4.17. Greenlet 0.4.16 works fine.

A temp fix is to uninstall greenlet 0.4.17 after installing gdbgui and then install greenlet 0.4.16.

I did:
```bash
pip3 install gdbgui
pip3 uninstall greenlet
pip3 install greenlet==0.4.16
```

I was then able to run gdbgui and open the dashboard.

**Comment by beaumanvienna** [comment]
Fixed it for me, awesome @SondreWang !

**Comment by cs01** [comment]
Thanks everyone, especially @SondreWang! This has been fixed in v0.14.0.1.
